modules/configuring-node-pools-for-hcp.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updates/updating-hosted-control-planes.adoc
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-node-pools-for-hcp_{context}"]
+= Configuring node pools for hosted control planes
+
+On hosted control planes, you can configure node pools by creating a `MachineConfig` object inside of a config map in the management cluster.
+
+//.Prerequisites
+
+//Are any prerequisites needed for this procedure? i.e., does the customer need to perform an update first?
+
+.Procedure
+
+. To create a `MachineConfig` object inside of a config map in the management cluster, enter the following information:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CONFIGMAP_NAME}
+  namespace: clusters
+data:
+  config: |
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: worker
+      name: ${MACHINECONFIG_NAME}
+    spec:
+      config:
+        ignition:
+          version: 3.2.0
+        storage:
+          files:
+          - contents:
+              source: data:...
+            mode: 420
+            overwrite: true
+            path: ${PATH}
```

Comment:
```suggestion
            path: ${PATH} <1>
```
This is up to you, but do you think its worth adding just a small description for this path object?

Reply:
Good question. I would need some expert guidance to know what to say about the path. @enxebre Any suggestions?